source/content/modules-plugins-known-issues.md

```diff
@@ -857,6 +857,29 @@ if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
 
 <hr />
 
+### [WPML - Automatic WPML registration using PHP for easy moves between development, test, and live environment](https://wpml.org/)
+**Issue**: Accessing /wp-admin/plugin-install.php?tab=commercial will throw `Sorry, you are not allowed to access this page`.
+
+**Solution**: Define WPML Site Key in wp-config.php 
+```php
+define( 'OTGS_INSTALLER_SITE_KEY_WPML', 'your-site-key' );
```

Comment:
@jahzlariosa Two questions:

1) Where does the user get the site key?
2) Is the site key something that's OK to have exposed in `wp-config.php`?

Reply:
1. Where does the user get the site key?

 - The site key is generated by the user from the plugin's website 
    https://wpml.org/account/sites/

2. Is the site key something that's OK to have exposed in wp-config.php?
- Yes it's ok :)